tool/modules/uds.py

```diff
@@ -184,32 +198,47 @@ def is_valid_response(message):
                 if is_valid_response(msg):
                     # Valid response
                     if verify:
-                        # Verification - backtrack the latest IDs and verify that the same response is received
+                        # Verification - backtrack the latest IDs and
+                        # verify that the same response is received
                         verified = False
-                        # Set filter to only receive messages for the arbitration ID being verified
-                        tp.set_filter_single_arbitration_id(msg.arbitration_id)
+                        # Set filter to only receive messages for the
+                        # arbitration ID being verified
+                        tp.set_filter_single_arbitration_id(
+                          msg.arbitration_id)
                         if print_results:
-                            print("\n  Verifying potential response from 0x{0:04x}".format(send_arbitration_id))
-                        verify_id_range = range(send_arbitration_id, send_arbitration_id - VERIFICATION_BACKTRACK, -1)
-                        for verification_arbitration_id in verify_id_range:
+                            print("\n  Verifying potential response from "
+                                  "0x{0:04x}".format(send_arb_id))
+                        verify_id_range = range(send_arb_id,
+                                                send_arb_id
+                                                - VERIFICATION_BACKTRACK,
+                                                -1)
+                        for verify_arb_id in verify_id_range:
                             if print_results:
-                                print("    Resending 0x{0:0x}... ".format(verification_arbitration_id), end=" ")
-                            tp.transmit(session_control_frames, verification_arbitration_id, None)
-                            # Give some extra time for verification, in case of slow responses
-                            verification_end_time = time.time() + delay + VERIFICATION_EXTRA_DELAY
+                                print("    Resending 0x{0:0x}... ".format(
+                                  verify_arb_id), end=" ")
+                            tp.transmit(session_control_frames,
+                                        verify_arb_id, None)
+                            # Give some extra time for verification, in
+                            # case of slow responses
+                            verification_end_time = time.time()
+                            + delay + VERIFICATION_EXTRA_DELAY
```

Comment:
This statement is now broken, as verification_end_time is simply set to `time.time()`. The delays on the next row are no longer added to this value.